index.html

```diff
@@ -3,8 +3,38 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <h1 class="header">
+            <p class="logo">My Logo</p>
+            <div><img class="menu" src="Union.png" alt="MenuBar"></div>
+        </h1>
+        <div class="notheader">
+            <div class="textcontainer">
+                <div class="primarytext">Hey There.<br>
+                    I'm Gabriel.
+                </div>
+                <div class="supportingtext">
+                    I'm a Software Developer
+                </div>
+                <div class="description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </div>
+            </div>
+            <ul class="socialmedia">
+                <li><img class="mediaicon" src="Disabled.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-1.png" alt="SociaMediaIcon"></li>            
+                <li><img class="mediaicon" src="Enabled.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-2.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-3.png" alt="SociaMediaIcon"></li>
+            </ul>
```

Comment:
Very thoughtful of you to have added the icons in an `li` tag, but I think one thing is missing which is finally adding the images in an `a` tag because they will be clickable and will serve as a way for your page visitors to reach out to you for an offer.